test_scripts/Defects/7_0/3542_apps_activation_after_resumption.lua

```diff
@@ -0,0 +1,93 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3542
+---------------------------------------------------------------------------------------------------
+-- Steps:
+-- 1. There are 2 MEDIA apps registered: App_1 and App_2
+-- 2. App_1 is in 'LIMITED' and App_2 is in 'BACKGROUND' HMI levels
+-- 3. Unexpected disconnect and reconnect are performed
+-- 4. SDL resumed HMI level of both apps:
+--  - App_1 is in 'LIMITED', App_2 is in 'BACKGROUND' HMI levels
+-- 5. App_2 activated
+-- 6. App_1 activated
+--
+-- SDL does:
+--  - change HMI level to 'FULL' for App_1
+--  - change HMI level to 'BACKGROUND' for App_2
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+config.defaultProtocolVersion = 2
+
+--[[ Apps Configuration ]]
+common.getConfigAppParams(1).appHMIType = { "MEDIA" }
+common.getConfigAppParams(2).appHMIType = { "MEDIA" }
+
+-- [[ Local Functions ]]
+local function unexpectedDisconnect()
+  common.getHMIConnection():ExpectNotification("BasicCommunication.OnAppUnregistered", { unexpectedDisconnect = true })
+  :Times(common.mobile.getAppsCount())
+  common.mobile.disconnect()
+  common.run.wait(1000)
+end
+
+local function startSession(pAppId)
+   common.getMobileSession(pAppId):StartRPC()
+end
+
+local function activateApp(pAppId)
+  local secondAppId = pAppId == 1 and 2 or 1
+  local cid = common.getHMIConnection():SendRequest("SDL.ActivateApp", { appID = common.getHMIAppId(pAppId) })
+  common.getHMIConnection():ExpectResponse(cid)
+  common.getMobileSession(pAppId):ExpectNotification("OnHMIStatus", { hmiLevel = "FULL" })
+  common.getMobileSession(secondAppId):ExpectNotification("OnHMIStatus", { hmiLevel = "BACKGROUND" })
+end
+
+local function deactivateApp(pAppId)
+  local secondAppId = pAppId == 1 and 2 or 1
+  common.getHMIConnection():SendNotification("BasicCommunication.OnAppDeactivated", { appID = common.getHMIAppId(pAppId) })
```

Comment:
The line length is greater than 120 characters, please update.

Reply:
@GetmanetsIrina Done in 0aa90d1